consul/src/main/scala/io/buoyant/consul/v1/ConsistencyMode.scala

```diff
@@ -0,0 +1,37 @@
+package io.buoyant.consul.v1
+
+import com.fasterxml.jackson.core.{JsonGenerator, JsonParser}
+import com.fasterxml.jackson.databind.{DeserializationContext, SerializerProvider}
+import io.buoyant.config.{ConfigDeserializer, ConfigSerializer}
+
+sealed trait ConsistencyMode
+
+object ConsistencyMode {
+
+  case object Default extends ConsistencyMode
+  case object Stale extends ConsistencyMode
+  case object Consistent extends ConsistencyMode
```

Comment:
we tend to just use `object` to `case object` -- it doesn't add anything valuable